### PR TITLE
Add Italian subtitles

### PR DESCRIPTION
**Pull request by relu91**
I used the same timings, they should be fine. I tested with windows video player and they look ok. Unfortunately, in some places, they are barely visible cause the video player displays the captions in white.

### CODE REVIEW

File: videos/WoT-Explainer-Video/WOTMotion.it.vtt [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+
+25
+00:01:34.500 --> 00:01:39.000
+e la distribuzione per browser,
+gli sviluppatori creano facilmente
+una soluzione IoT full stack.
+
+26
+00:01:39.500 --> 00:01:42.000
+La loro suluzione si connette
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.it.vtt:136]: soluzione

File: videos/WoT-Explainer-Video/WOTMotion.it.vtt [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+20
+00:01:15.500 --> 00:01:19.000
+ma può essere anche applicato 
+al problema del nostro frutticoltore 
+in ambito agricolo. 
+
+21
+00:01:19.500 --> 00:01:23.000
+Gli sviluppatori sono felici
+di trovare un lista
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.it.vtt:111]: una liste

File: videos/WoT-Explainer-Video/WOTMotion.it.vtt [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+
+33
+00:02:08.500 --> 00:02:13.000
+come pagine HTML, pacchetti JavaScript o
+file CSS presi da progetti Web precedenti.
+
+34
+00:02:13.500 --> 00:02:18.000
+L'elemento principale della
+methodologia WoT
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.it.vtt:178]: metodologia

File: videos/WoT-Explainer-Video/WOTMotion.it.vtt [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+index.html per le Thing.
+
+37
+00:02:21.500 --> 00:02:30.000
+Molte informazioni possono
+essere ottenute dal TD
+
+38
+00:02:30.500 --> 00:02:40.000
+come: l'idenditificazione, l'organizzazione dei dati,
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.it.vtt:197]: l'identificazione

File: videos/WoT-Explainer-Video/WOTMotion.it.vtt [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+
+38
+00:02:30.500 --> 00:02:40.000
+come: l'idenditificazione, l'organizzazione dei dati, 
+modalità d'accesso, unità di misura, tipi, modi di interazione, 
+sicurezza o relazioni. 
+
+39
+00:02:40.500 --> 00:02:44.000
+I progetti IoT classici usalmente
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.it.vtt:203]: usualmente ?

File: videos/WoT-Explainer-Video/WOTMotion.it.vtt [hunk 6]
```diff
@@ -0,0 +1,263 @@
...
+realizzato. 
+
+49
+00:03:22.500 --> 00:03:27.000
+Ora gli sviluppatori sono pronti ad 
+affrontare la loro prossima sfida IoT. 
+
+50
+00:03:27.500 --> 00:03:41.000
+E certamente sarà WoT LA methodologia
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.it.vtt:261]: metodologia ?

**Comment by relu91** [comment]
Thank you, Daniel! sorry but I couldn't make work a spell checker in my code editor for Italian 🤣

**Review by danielpeintner — APPROVED** [review]
LGTM

File: videos/WoT-Explainer-Video/WOTMotion.it.vtt [hunk 7]
```diff
@@ -0,0 +1,263 @@
...
+42
+00:02:51.500 --> 00:02:57.000
+Al contrario, grazie a WoT gli sviluppatori
+possono creare soluzioni IoT in poche
+sessioni di sviluppo
+
+43
+00:02:57.500 --> 00:03:03.000
+combinando i WoT TD, protocol binding,
+e API di scripting alla loro conoscenze
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.it.vtt:225]: "alla loro conoscenze" 
-->
"alla loro conoscenza" ??

Comment (danielpeintner) [reply videos/WoT-Explainer-Video/WOTMotion.it.vtt:225]: @relu91 ?

**Review by danielpeintner — APPROVED** [review]

**Pull request merged**